recipes-security/optee-imx/optee-os/0001-optee-os-fix-gcc10-compilation-issue-and-missing-cc-.patch

```diff
@@ -89,7 +93,7 @@ index c05815f3..68f45552 100644
  
  PLATFORM_$(PLATFORM) := y
 @@ -16,10 +17,6 @@ PLATFORM_FLAVOR_$(PLATFORM_FLAVOR) := y
- $(call cfg-depends-all,CFG_PAGED_USER_TA,CFG_WITH_PAGER CFG_WITH_USER_TA)
+ $(eval $(call cfg-depends-all,CFG_PAGED_USER_TA,CFG_WITH_PAGER CFG_WITH_USER_TA))
```

Comment:
Why use this `eval` here? Doesn't the old `call` work?

Reply:
This is done in the upstream with commit [6b385aaf7a1e83b689e90a4c4290d5406b172ca2](https://github.com/jforissier/optee_os/commit/6b385aaf7a1e83b689e90a4c4290d5406b172ca2)

NXP updated its fork, so that commit got integrated as well.

What I did in here is I re-based the patch to the latest version in the fork, hence you see a bit more differences here. 😉

Reply:
Ol, cool.